test/frontend/reducers/reducer_authentication.spec.js

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
```

Comment:
Do we need to use this since jasmine injects its own version of `expect` into test files?